src/Components/SideBar/SideBar.tsx

```diff
@@ -0,0 +1,28 @@
+import { NavLink } from "solid-app-router";
+import type { Component } from "solid-js";
+import { navSignal } from "../../Store/side-bar-store";
+import { For } from "solid-js";
+
+export const SideBar: Component = () => {
+  const [navs, _] = navSignal;
+  return (
+    <div class="flex flex-col max-w-xs w-64 fixed h-screen z-10 pl-4 border-gray-100 border-r-2">
+      <p class="flex relative justify-start items-center text-4xl mb-6 ml-1 font-bold text-gray-800">
+        Deriv API
+      </p>
+      <For each={navs()}>
+        {(item, index) => (
+          <NavLink
+            data-index={index}
+            activeClass="before:sidebar-after bg-gray-100 font-bold text-black"
+            class="flex relative justify-start items-center py-2.5 pr-4 pl-4 mb-4 text-base rounded-l-md h-11 text-gray-500 hover:bg-gray-100 hover:before:sidebar-before"
```

Comment:
Ok, so I understand that here you define hover:before: and I wonder if tailwind handles from the css https://github.com/mamali-fs/api-deriv-com-solidjs/pull/16/files#diff-85da366246340e911d7a0eb9153e64137a1b31f0686c74d9aef9ae4f032cb09eR70 could be used instead of class

Reply:
In that case, I had to repeat `hover:before:` multiple times, so I used class instead😁

Reply:
sure, so I see that for repeating styling with classes is still useful, nice